frontend/public/components/notification-drawer.tsx

```diff
@@ -179,6 +188,27 @@ const getUpdateNotificationEntries = (
       />,
     );
   }
+  if (failedPlugins.length > 0) {
+    entries.push(
+      ...failedPlugins.map((plugin) => {
+        const link = resourcePath(referenceForModel(ConsolePluginModel), plugin.pluginName);
+        return (
+          <NotificationEntry
+            actionPath={link}
+            actionText={i18next.t('public~View plugin')}
+            key={`${plugin.pluginName}-dynamic-plugin-fail`}
+            description={i18next.t('public~Something went wrong with the {{pluginName}} plugin.', {
```

Comment:
OK as a follow up, but I think we should try to capture the original error to help admins debug. For instance, if there is an HTTP error loading the plugin manifest, we can add that to the status in the plugin store and show it here.

cc @vojtechszocs

Reply:
This should mirror whatever in the dashboard status popup too